src/mem/freelist.h

```diff
@@ -216,16 +261,16 @@ namespace snmalloc
    *
    * Checks signing of pointers
    */
+  template<SNMALLOC_CONCEPT(capptr_bounds::c) B>
```

Comment:
I don't think I understand this code or what the template parameter does.  I expect this to be providing an iterator that returns domesticated pointers, over a list that may store wild pointers (and where the head of the list is domestic?)?